src/commcare_cloud/ansible/roles/postgresql/templates/pg_hba.conf.j2

```diff
@@ -61,3 +61,11 @@ host    all             all             127.0.0.1/32            md5
 
 # IPv6 local connections:
 host    all             all             ::1/128                 md5
+
+# should accept connection from pgpool host if applicable
+{% for pgpool_hostname, config in pgpool_config.items() %}
+{% set pgpool_host = pgpool_hostname | host_from_alias(deploy_env) %}
```

Comment:
I think you could use `groups[pgpool_hostname]` to get a list containing a single string that matches `pgpool_host` (probably an IP address).

Also I think you should use a term like "host_group" instead of `hostname` because `hostname` usually refers to the linux hostname which may be different.

Reply:
>  I think you could use groups[pgpool_hostname]
That's a good idea, that would be equivalent to what `translate_host` does which will need `len(group) == 1` validation though. Using translate_host made it bit cleaner.

> should use a term like "host_group" instead of hostname
That way with below example we would be calling `db1` a host group but we don't mean so. So, I named it that way, but true it could get confusing, 

```
[db1]
192.168.33.16
```

Reply:
This is now removed completely